thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/alert/scheme/DetectionEmailAlerter.java

```diff
@@ -198,15 +202,15 @@ private void sendEmail(DetectionAlertFilterRecipients recipients, Set<MergedAnom
         new EmailContentFormatterContext());
 
     HtmlEmail email = emailEntity.getContent();
+    email.setSubject(emailEntity.getSubject());
     email.setFrom(this.config.getFrom());
     email.setTo(AlertUtils.toAddress(recipients.getTo()));
-    email.setSubject(emailEntity.getSubject());
-    if (CollectionUtils.isNotEmpty(recipients.getCc())) {
-      email.setCc(AlertUtils.toAddress(recipients.getCc()));
-    }
-    if (CollectionUtils.isNotEmpty(recipients.getBcc())) {
-      email.setBcc(AlertUtils.toAddress(recipients.getBcc()));
+    if (recipients.getCc() == null) {
```

Comment:
if Cc() is mandatory then it should get initialized as empty set in constructor.

Reply:
It is not mandatory but I think it makes sense to initialize them in the constructor itself.

Reply:
On second thought, I'd rather keep it this way. cc and bcc are optional fields.

Reply:
Makes sense.